internal/sirius/edit_team.go

```diff
@@ -1,35 +1,46 @@
 package sirius
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"net/url"
-	"strconv"
-	"strings"
 )
 
+type editTeamRequest struct {
+	Name        string `json:"name"`
+	Type        string `json:"type"`
+	Email       string `json:"email"`
+	PhoneNumber string `json:"phoneNumber"`
+	MemberIds   []int  `json:"memberIds"`
+}
+
 func (c *Client) EditTeam(ctx Context, team Team) error {
-	form := url.Values{
-		"name":             {team.DisplayName},
-		"email":            {team.Email},
-		"phoneNumber":      {team.PhoneNumber},
-		"teamType[handle]": {team.Type},
+	memberIds := []int{}
```

Comment:
or since you know how many members

```go
memberIDs := make([]int, len(team.Members))
for i, member := range team.Members {
  memberIDs[i] = member.ID
}
```

Reply:
Done, good to make use of the language properly 👍🏽